app/src/main/res/values/strings.xml

```diff
@@ -671,6 +684,11 @@
     <string name="settings_theme_dialog_themes_3">Dark</string>
     <string name="settings_theme_dialog_themes_4">True Black</string>
 
+    <string name="settings_notification_dialog_title">Notification Action</string>
+    <string name="settings_notification_dialog_option_1">No action button</string>
+    <string name="settings_notification_dialog_option_2">STOP button</string>
```

Comment:
do not use upper-case letters except in command buttons.

Reply:
Modified.